servicecontrol/transport-adapter.md

```diff
@@ -0,0 +1,110 @@
+---
+title: ServiceControl Transport Adapter
+summary: placeholder
+component: SCTransportAdapter
+---
+
+The ServiceControl Transport Adapter decouples ServiceControl from the specifics of the business endpoint transport. 
```

Comment:
I think here we should explain briefly (1 paragraph) why this component exists in the first place, what problems does it solve?

e.g. Transport Adapter supports transport-specific features/certain configurations? for SQL Server, RabbitMQ and ASB transports, to ensure that all SC functionality is supported for any configuration of those transports. e.g. bla bla